PyInstaller/utils/win32/winresource.py

```diff
@@ -202,7 +202,7 @@ def UpdateResources(dstpath, data, type_, names=None, languages=None):
             for language in res[type_][name]:
                 logger.info("Updating resource type %s name %s language %s",
                             type_, name, language)
-                win32api.UpdateResource(hdst, type_, name, data, language)
+                win32api.UpdateResource(hdst, type_, name, bytes(data, 'UTF-8'), language)
```

Comment:
Can you please explain this to me, I'm not a windows-guy. https://msdn.microsoft.com/en-us/library/windows/desktop/ms648049%28v=vs.85%29.aspx says: "strings or text must be in Unicode format", but what is "Unicode format"?